packages/calcite-components/src/components/input-message/input-message.scss

```diff
@@ -4,33 +4,35 @@
  * These properties can be overridden using the component's tag as selector.
  *
  * @prop --calcite-input-message-spacing-value: The top margin spacing above the component.
+ * @prop --calcite-input-message-spacing: Specifies the top margin space above the component.
```

Comment:
Should this spacing only work for the space above? If so, should it be named accordingly? I think the name and the description don't really match right now.

What is the difference between `--calcite-input-message-spacing-value` and `--calcite-input-message-spacing`? I think these are confusing.

Reply:
Agreed - just one here should suffice - it should only apply to the top spacing. The old one should probably be deprecated, "spacing-value" is a strange nomenclature for a public property.